docs/Pitfalls.asciidoc

```diff
@@ -36,3 +36,20 @@ user can share the environment without problems.
 This approach will lead to a problem when the openqa package is updated, since the
 directory permissions will be changed again, nothing a `chmod -R g+rwx /var/lib/openqa/`
 and `chgrp -R openqa /var/lib/openqa` can not fix.
+
+== Performance impact
+
+openQA workers can cause high I/O load, especially when creating VM snapshots.
+The impact therefore gets more severe when +MAKETESTSNAPSHOTS+ is enabled.
+should not impact the stability of openQA jobs but can increase job execution
+time. If you run jobs on a machine where responsiveness of other services
+matter, for example your desktop machine, consider patching the
+IOSchedulingPriority of a workers service file as described in the
+https://www.freedesktop.org/software/systemd/man/systemd.exec.html#IOSchedulingPriority=[systemd
+documentation], for example set +IOSchedulingPriority=7+ for the best-effort
```

Comment:
If you want best-effort you need to set IOSchedulingClass=best-effort and not the priority

Reply:
isn't best-effort the default scheduling class? and if it is then my intention was to set the priority only within the default class

Reply:
then just remove the mentioning of the IO scheduling class?

Reply:
yes, sir!